apps/web-giddh/src/app/audit-logs/components/audit-logs-grid/audit-logs-grid.component.html

```diff
@@ -29,133 +29,156 @@ <h1 class="audit-log-heading">{{log.operationType}}
         <div *ngIf="log.log && log.log?.sharedWith && log.log?.sharedWith != undefined && log.log?.sharedWith?.name.length > 0">
             <label class="pull-left mr-l2">{{localeData?.shared_with}}:</label>
             <p class="pull-left mr-l05">{{log.log?.sharedWith?.name}}
-                <span *ngIf="log.log?.sharedWith?.email && log.log?.sharedWith?.email != null && log.log?.sharedWith?.email?.length > 0">
+                <span
+                    *ngIf="log.log?.sharedWith?.email && log.log?.sharedWith?.email != null && log.log?.sharedWith?.email?.length > 0">
                     | {{log.log?.sharedWith?.email}}</span>
             </p>
         </div>
         <div *ngIf="log && log.log && log.log?.description != undefined && log.log?.description?.length > 0">
             <label class="pull-left mr-l2">{{localeData?.description}}:</label>
             <p class="pull-left mr-l05">{{log.log?.description}}</p>
         </div>
-    </div>
+        <div class="mr-t05 table-container" *ngIf="log.log != null && log.log?.transactions != undefined && log.log?.transactions?.length > 0">
+            <table class="table table-bordered ">
+                <thead>
+                    <tr>
+                        <th class="col-3 text-center">#</th>
```

Comment:
@shalinee-walkover can we remove this col-3, because there is no row given and we've already used th, there should be no use or col-3. Also, 5 col-3 would be col-15 which is not possible

Reply:
I am not sure about it, also I have to do unit testing if I do change. I think we have to Let it be like this

Reply:
@shalinee-walkover is it working fine if we let the col as it is??